### PR TITLE
fix(DB/Creature): Rebuild Old Hillsbrad critters

### PR DESCRIPTION
**Pull request by Gultask**
WIP: Need to find more guids

## Changes Proposed:
-  Sniffs critters for Old Hillsbrad, corrects wander distance and spawn timer

## Issues Addressed:

- Closes 

## SOURCE:

Retail sniffs

## Tests Performed:

- Tested ingame
- 

## How to Test the Changes:

1. `.tele OldHillsbrad`
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
I'll rather rebuild the spawns. They're all over the place, not literally.

File: data/sql/updates/pending_db_world/rev_1673668700960590200.sql [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
...3532,83533,83534,83535,83536,83537,83538,83539,83540,83541,83542,83543,83544,83545,83546,83547,83548,83549,83550,83551,83552,83553,83554,83555,83556,83557,83558,83559,83560,83561,83562,83563,83564,83565,83566,83567,83568,83569,83570,83571,83572,83573,83574,83575,83576,83577,83578,83579,83580,83581,83582,83583,83584,83585,83586,83587,83588,83589,83590,83591,83592,83593,83594,83595,83596,83597,83598,83599,83600,83601,83602,83603,83604,83605,83606,83607,83608,83609,83610,83611,83612,83613,83614,83615,83616,83617,83618,83619,83620,83621,83622,83623,83624,83625,83626,83654,83655,83656,83657,83658,83659,83660,83661,83662,83663,83664,83665,83666,83667,83732,83733,83734,83735,83736,83737,83738,83739,83740,83741,83742,83743,83744,83745,83746,83747,83748,83749,83750,83751,83752,83753,83754,83755,83756,83757,83758,83759,83760,83761,83762,83763,83764,83765,83766,83767,83768,83769,83770,83771,83772,83773,83774,83775,83776,83777,83778,83779,83780,83781,83782,83783,83784,83785,83786,83787,83788,83789,83790,83791,83792,83793,83794,83795,83796,83797,83798,83799,83800,83801,83802,83803,83804,83805,83806,83807,83808,83809,83810,83811,83812,83813,83814,83815,83816,83817,83818,83819,83820,83821,83822);
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1673668700960590200.sql:3]: why is DELETE FROM `creature` WHERE `map`=560 AND `id1` IN (2385, 2350, 2354, 2408) AND `guid` IN  repeated in both lines?

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1673668700960590200.sql (hunk 1 above)
Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1673668700960590200.sql:3]: One is for the old creatures, the other is for the INSERT ones.
I should've specified in comments, too lazy 😛

Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1673668700960590200.sql:3]: Is this good to go for approval?

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1673668700960590200.sql:3]: ye

**Review by acidmanifesto — APPROVED** [review]

**Pull request merged**